Trend Micro/trend-micro-apex-one/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,294 @@
+      },
+      {
+        "field": "source.ip"
+      }
+    ]
+  },
+  {
+    "value": "Engine '{Trend_Micro.Engine}' on {host.name} has status: {Trend_Micro.Engine_Status}.",
+    "conditions": [
+      {
+        "field": "Trend_Micro.Engine"
+      },
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "Trend_Micro.Engine_Status"
+      }
+    ]
+  },
+  {
+    "value": "Authentication event at {host.name} for process '{process.title}' with reason: {event.reason}.",
+    "conditions": [
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "process.title"
+      },
+      {
+        "field": "event.reason"
+      }
+    ]
+  },
+  {
+    "value": "Network event from {source.ip}:{source.port} to {destination.ip}:{destination.port} for process '{process.executable}'. Threat: {Trend_Micro.NCIE_ThreatName}. Action: {event.action}.",
+    "conditions": [
+      {
+        "field": "source.ip"
+      },
+      {
+        "field": "destination.ip"
+      },
+      {
+        "field": "process.executable"
+      },
+      {
+        "field": "Trend_Micro.NCIE_ThreatName"
+      },
+      {
+        "field": "event.action"
+      }
+    ]
+  },
+  {
+    "value": "Configuration change on {host.name} ({host.ip}).",
+    "conditions": [
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "host.ip"
+      }
+    ]
+  },
+  {
+    "value": "Predictive ML Alert on {host.ip}. Executed by {user.name}.",
+    "conditions": [
+      {
+        "field": "host.ip"
+      },
+      {
+        "field": "user.name"
+      }
+    ]
+  },
+  {
+    "value": "Sandbox Alert on {host.name}, threat originates from {url.original}.",
+    "conditions": [
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "url.original"
+      }
+    ]
+  },
+  {
+    "value": "Spyware/Grayware Alert on {host.name}. Detected {Trend_Micro.VirusName}. File involved: {file.name}. Threat originates from IP {source.ip}.",
+    "conditions": [
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "Trend_Micro.VirusName"
+      },
+      {
+        "field": "file.name"
+      },
+      {
+        "field": "source.ip"
+      }
+    ]
+  },
+  {
+    "value": "Suspicious file alert on {host.name}. File: {file.name} with SHA1 hash: {file.hash.sha1} from IP: {source.ip}. Action taken: {event.action}.",
+    "conditions": [
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "file.name"
+      },
+      {
+        "field": "file.hash.sha1"
+      },
+      {
+        "field": "source.ip"
+      },
+      {
+        "field": "event.action"
+      }
+    ]
+  },
+  {
+    "value": "Virus/Malware detected on {host.name}, User: {user.name}. Origin IP: {source.ip}.",
+    "conditions": [
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "user.name"
+      },
+      {
+        "field": "source.ip"
+      }
+    ]
+  },
```

Comment:
I think that the smart-description L199 (ie for Predictive Machine learning) should work for virus and malware events (must be checked).
```suggestion
```